document/js-api/index.bs

```diff
@@ -93,10 +93,10 @@ urlPrefix: https://webassembly.github.io/spec/core/; spec: WebAssembly; type: df
     text: trap; url: exec/runtime.html#syntax-trap
     url: exec/runtime.html#values
         text: WebAssembly value
-        text: 𝗂𝟨𝟦.𝖼𝗈𝗇𝗌𝗍
-        text: 𝗂𝟥𝟤.𝖼𝗈𝗇𝗌𝗍
-        text: 𝖿𝟥𝟤.𝖼𝗈𝗇𝗌𝗍
-        text: 𝖿𝟨𝟦.𝖼𝗈𝗇𝗌𝗍
+        text: 헂ퟨퟦ.햼허헇헌헍
```

Comment:
o_O

What happened here? :-)

Reply:
Wow, no idea. I think my editor barfed on the extended Unicode. Fixed, but @littledan, why this fancy? I for one have no idea how I would even input these characters if I needed to edit any of this.